### PR TITLE
[CR] Tweaked @-menu (player stats description submenu) UI

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: Interface "Tweaked @-menu (player stats description submenu) UI."```

#### Purpose of change
Fix translated strings not fitting; color coding; don't show some irrelevant information.

#### Describe the solution
- Got rid of fixed x coordinate as a starting coordinate to begin description text - alignment only worked for English language (and not always correctly) and was broken for most other languages. 
- Merged printing of two strings into one, replaced `mvwprintz` with `print_colored_text`.
- Made description text light gray instead of generic magenta, and value text white.
- Used `weight_units()` function instead of a check for `USE_METRIC_WEIGHTS`.
- Added a check for showing skill rust.

### Before:
Incorrect alignment for strength > 10 and pounds
изображение

Incorrect alignment for ranged penalty
изображение

Needless skill rust value even if skill rust is disabled
изображение

Translated strings don't fit
изображение

### After:
New color coding, no artificial alignment (= works correctly for all languages)
изображение

Show skill rust percentage with skill rust enabled
изображение

Don't show skill rust percentage if skill rust is disabled.
изображение

### CODE REVIEW

File: src/player_display.cpp [hunk 1]
```diff
@@ -268,52 +268,52 @@ static void draw_stats_tab( const catacurses::window &w_stats, const catacurses:
...
-        mvwprintz( w_info, 3, 38, c_magenta, "%+.1lf", you.get_hit_base() );
-        mvwprintz( w_info, 4, 1, c_magenta, _( "Ranged penalty:" ) );
-        mvwprintz( w_info, 4, 38, c_magenta, "%+3d", -( abs( you.ranged_dex_mod() ) ) );
-        mvwprintz( w_info, 5, 1, c_magenta, _( "Throwing penalty per target's dodge:" ) );
-        mvwprintz( w_info, 5, 38, c_magenta, "%+3d", you.throw_dispersion_per_dodge( false ) );
+        print_colored_text( w_info, 3, 1, c_light_gray, c_light_gray,
+                            string_format( _( "Melee to-hit bonus: <color_white>%+.1lf</color>" ), you.get_hit_base() ) );
+        print_colored_text( w_info, 4, 1, c_light_gray, c_light_gray,
+                            string_format( _( "Ranged penalty: <color_white>%+d</color>" ),
+                                           -( abs( you.ranged_dex_mod() ) ) ) );
```
Comment (BevapDin) [inline src/player_display.cpp]:
```suggestion
                                           -abs( you.ranged_dex_mod() ) ) );
```

**Comment by Night-Pryanik** [comment]
Could anyone help me understand what are gorgon and travis complaining on? My VS doesn't generate any such errors.

**Comment by OzoneH3** [comment]
```
Problem in Semantic analysis: No matching function for call to 'print_colored_text'

Hint: Candidate function not viable: expects an l-value for 4th argument
See: void print_colored_text (const catacurses::window&, int, int, nc_color&, const nc_color&, const std::__cxx11::string&) in output.h :229
```

For some reason, the first color variable in the print_colored_text function is not const ref.
You'll have to do something like this, or change the function variable to const. (Which would require a thorough investigation if this is actually used somewhere for some reason)

```
nc_color col_temp = c_light_gray;
print_colored_text( w_info, 3, 1, col_temp, c_light_gray,
                            string_format( _( "Melee to-hit bonus: <color_white>%+.1lf</color>" ), you.get_hit_base() ) );
```

**Comment by JeanLucVanDamme** [comment]
I'm not convinced that removing the alignment of values in the center is a positive usability change.

Alignment helps you scan through the values quickly. That's partly why it's used in every other section (stats, skills, etc). The change makes scanning more difficult.

On the other hand, I see how it addresses display inconsistencies and other bugs. And it actually feels a little prettier and breaks up the monotony of this screen.

Either way, thanks for beginning some much-needed UI work :)

**Comment by Night-Pryanik** [comment]
> alignment of values in the center

The problem is that it doesn't even aligned by the center 
изображение

As an alternative I tried to make all values for all stats align at some fixed arbitrary number (higher than the current one), identical for all stats, something like that
изображение

But I didn't find it better than my first variant, as there are too many free space.

**Comment by JeanLucVanDamme** [comment]
Ah, I didn't realize how alignment-resistant this is. Your solution is a big improvement, and the best of these options.

**Comment by AMurkin** [comment]
You prob can use `printf` padding:
```cpp
        print_colored_text( w_info, 3, 1, col_temp, c_light_gray,
                            string_format( _( "%-20s <color_white>%5d</color>" ), _( "Base HP:" ), you.hp_max[1] ) );
        print_colored_text( w_info, 4, 1, col_temp, c_light_gray,
                            string_format( _( "%-20s <color_white>%5.1f</color>" ), string_format( _( "Carry weight (%s):" ), weight_units() ),
                                           convert_weight( you.weight_capacity() ) ) );
        print_colored_text( w_info, 5, 1, col_temp, c_light_gray,
                            string_format( _( "%-20s <color_white>%5.1f</color>" ), _( "Melee damage:" ), you.bonus_damage( false ) ) );
```

STR

**Comment by Night-Pryanik** [comment]
> You prob can use printf padding:

Setting arbitrary numbers for the length of description text could leave the problem of translated strings not fitting unresolved.

**Comment by AMurkin** [comment]
But you could leave the template strings marked for translation and these numbers could be changed along with the title strings.

**Comment by Night-Pryanik** [comment]
You mean something like `translate_context( "9 symbols", "base hp: " )`? (I don't remember exact syntax).

**Comment by AMurkin** [comment]
I mean `_( "%-20s <color_white>%5d</color>" )` is translatable string and translators can change the number when they want.
And source could be: `pgettext( "20 is padding width for the title, 5 is padding width for the value", "%-20s <color_white>%5d</color>" )`

**Pull request merged**